pegasus/sites.v3/hourofcode.com/styles/060-hoc-specific.css

```diff
@@ -1,24 +1,78 @@
 #hoc-header-branding {
-    width: 105px;
-    margin: -1.625em 0 0 0;
-    padding: 0;
-    border-left: 2px solid #fff;
-    border-right: 2px solid #fff;
+  width: 8em;
+  position: relative;
+  top: -2em;
+  left: 0;
+  float: left;
 }
 #hoc-header-branding img {
-    width: 100%;
-    max-width: 100%;
+  width: 100%;
+  max-width: 100%;
+}
+#hoc-header-main {
+  margin-top: 1em;
+  text-align: right;
+}
+#language-dropdown {
+  float: right;
+  margin-left: 2em;
```

Comment:
The height of the teal horizontal header bar was defined in em originally (before this change), and I wanted to position the branding logo vertically centered with this bar, which is why I used em here, to be consistent with the units of the header bar itself.